src/main/java/org/folio/orders/utils/ResourcePathResolver.java

```diff
@@ -37,6 +39,7 @@ private ResourcePathResolver() {
     apis.put(PIECES, "/orders-storage/pieces");
     apis.put(RECEIVING_HISTORY, "/orders-storage/receiving-history");
     apis.put(PO_LINE_NUMBER, "/orders-storage/po-line-number");
+    apis.put(SEARCH_ORDER, "/orders-storage/orders");
```

Comment:
Really not important but maybe make SEARCH_ORDER plural (SEARCH_ORDERS)

Reply:
done